stack/language_template.go

```diff
@@ -50,6 +51,8 @@ func ParseYAMLDataForLanguageTemplate(fileData []byte) (*LanguageTemplate, error
 func IsValidTemplate(lang string) bool {
 	var found bool
 
+	lang = strings.ToLower(lang)
```

Comment:
@rgee0 Can we create an issue stating that we should write a docs page for docs.openfaas.com describing how templates work and how to create one? It should be based upon the existing markdown in the faas-cli repo.